CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ### 6.0 [not yet released]
+- When using a DecimalEncodedValue with useMaximumAsInfinity=true and a single bit of space make sure you always use 
+  Double.POSITIVE_INFINITY to set the value, see #2646
```

Comment:
Not just if only a single bit of space is used, right? DecimalEncodedValueImpl now uses 0 as default value, and if we want it to return something else we need to set the value explicitly(?).